### DISCUSSION TITLE
Loader when listener changed

### DISCUSSION THREAD

**Discussion by mucan54** [orchidsoftware/platform#1873] [Ideas]
I'm using listener to load data based on listener value but it takes some times to load data, it waits and user can not figure out if it is loading. I think there should appear a loader icon while listener loads data.

**Comment by tabuna** [comment]
Hi @mucan54, we are using https://turbo.hotwired.dev/handbook/drive#displaying-progress , showing a download line at the top of the page for requests that are more than 500ms. Should I set a lower value? Or don't you see this streak?

**Reply by mucan54** [reply]
it doesn't appear while listener loads.

**Reply by lucatrev** [reply]
hi @tabuna,
i was looking for same feature and i can confirm no feedback is given to user during listener data loading, any hints to solve?

@mucan54 did you find any workaround to solve?
